doc/classes/Window.xml

```diff
@@ -120,6 +120,32 @@
 				Returns layout direction and text writing direction.
 			</description>
 		</method>
+		<method name="get_output_max_linear_value" qualifiers="const">
+			<return type="float" />
+			<description>
+				Returns the maximum value for linear color components that can be displayed in this window, regardless of SDR or HDR output. This method will return 1.0 if HDR is not enabled or not supported.
+				Use this to scale content to max out the display's brightness, for example lasers or other bright effects. The following is an example that produces the brightest red color that the display can produce:
+				[codeblocks]
+				[gdscript]
+				var bright_red: Color = Color.RED
+				# Color must be linear-encoded to use math operations correctly.
+				bright_red = bright_red.srgb_to_linear()
+				# Record the original alpha value.
+				var original_alpha: float = bright_red.a
+				# Adjust brightness to be the brightest possible, regardless of SDR or HDR output.
+				bright_red = bright_red * get_window().get_output_max_linear_value()
+				# Restore the original alpha channel value.
+				bright_red.a = original_alpha
+				# Convert back to nonlinear sRGB encoding.
+				bright_red = bright_red.linear_to_srgb()
+				[/gdscript]
+				[csharp]
+				// TODO
+				[/csharp]
```

Comment:
If you got nothing strip this out entirely. There are no TODOs in the class reference XMLs anymore.
```suggestion
```